docs/agents/hand-agent.rst

```diff
@@ -18,39 +18,42 @@ Control Schemes
 ---------------
 
 - **Raw Joint Torques** (``0``)
-  
+
   23 length vector of raw torques to pass into the joints, in the order listed
   beow in :ref:`hand-joints`
 
 - **Scaled Joint Torques** (``1``)
-  
+
   23 length vector of scaled torques, between ``-1`` and ``1``. The strength
   finger each joint is scaled depending on the weight of the bone and if it is
   a finger or not. ``1`` represents the maximum power in the forward direction
 
 
 - **Scaled Joint Torques + Floating** (``2``)
-  
+
   Same as above, but the vector is of length 26, with the last three values
   representing the amount of movement in the ``[x, y, z]`` directions (see
   :ref:`coordinate-system`), with a maximum of ``0.5`` meters of freedom.
 
+  The last coordinates allow the HandAgent to float around.
+
+
 .. _`hand-joints`:
 
 HandAgent Joints
 ----------------
 The control scheme for the HandAgent and the
-:class:`~holodeck.sensors.JointRotationSensor` use a 94 length vector refer 
```

Comment:
Any particular reason for the removal of tailing spaces? I approve of it but am curious if it affects formatting.

Reply:
The linter yells at you when you have trailing whitespace. There is a VSCode command to trim them.